pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -2245,7 +2248,7 @@ private void createPendingLoadTopic() {
 
         final String topic = pendingTopic.getLeft();
         try {
-            checkTopicNsOwnership(topic);
+            checkTopicNsOwnership(topic).join();
```

Comment:
Instead of blocking here, we should chain on the completion of this future

Reply:
Done.